### PR TITLE
landing page coins list, charts, nav, theme and search

### PR DESCRIPTION
**Pull request by oerkperl**

### CODE REVIEW

File: app/components/home/navigation/MarketData.tsx [hunk 1]
```diff
@@ -51,20 +27,22 @@ export const MarketData = () => {
...
+      const { data } = await axios("https://api.coingecko.com/api/v3/global");
 
-        setMarketData(data);
-        if (data) {
-          setIsLoading(false);
-        }
-      } catch (err) {
-        console.error(err);
+      setMarketData(data);
+      console.log(data);
```
Comment (yoloOnTheBattlefield) [inline app/components/home/navigation/MarketData.tsx:35]: no console

File: app/globals.css [hunk 2]
```diff
@@ -16,12 +16,15 @@
...
+[data-theme='light'] {
+  background-color: #ffffff;
+  color: #333333;
 }
+
+[data-theme='dark'] {
+  background-color: #333333;
+  color: #ffffff;
+} */
+
```
Comment (yoloOnTheBattlefield) [inline app/globals.css:30]: remove this

File: app/lib/utils/components/ProgressBar.tsx [hunk 3]
```diff
@@ -1,25 +1,47 @@
...
+  useEffect(() => {
+    const fg = theme === "dark" ? "#5EFF5A" : "#01F1E3";
+    downTrend ? setFg("#E323FF") : setFg(fg);
+  }, [theme]);
+
   return (
-    <Container width={width}>
-      <Bar progress={progress} />
+    <Container $width={width}>
+      <Bar $progress={progress} $foregroundColor={fg} />
```
Comment (yoloOnTheBattlefield) [inline app/lib/utils/components/ProgressBar.tsx:44]: do we really need state?

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/lib/utils/components/ProgressBar.tsx (hunk 3 above)
Comment (oerkperl) [reply app/lib/utils/components/ProgressBar.tsx:44]: I have tried this but it is not producing the desired effect. I am open for suggestion 
export const ProgressBar: React.FC<{
  progress: string;
  width?: string;
  downTrend?: boolean;
}> = ({ progress, width, downTrend }) => {
  const { theme } = useTheme();
  let color = theme === "dark" ? "#5EFF5A" : "#01F1E3";

  useEffect(() => {
    if (downTrend) {
      color = "#E323FF";
    }
  }, [theme, downTrend]);

  return (
    <Container $width={width}>
      <Bar $progress={progress} $foregroundColor={color} />
    </Container>
  );
};

File: app/globals.css (hunk 2 above)
Comment (oerkperl) [reply app/globals.css:30]: ok

File: app/components/home/navigation/MarketData.tsx (hunk 1 above)
Comment (oerkperl) [reply app/components/home/navigation/MarketData.tsx:35]: ok

**Pull request merged**